### ISSUE TITLE
Thoughts on shows sharing the default category taxonomy

### ISSUE THREAD

**Issue by superlou** [trms/wp-cablecast#23]
I wanted to get an understanding of how everyone is using the category taxonomy with show post types. I'm running into an issue that most of the time, users either want to see all shows in a category, or all Wordpress posts in a category, but not usually both. Since the built-in URL for a category doesn't discriminate between post types, a category listing will have mix of posts and shows. There also is some confusion for editors as to why there are some similar looking categories available for posts that are only used for shows.

Has anyone found an elegant way in Wordpress to get a category listing for just shows? Would anyone be interested in having a custom taxonomy, e.g. `show-category` that is specific to the automatically syncing content coming from Cablecast?

**Comment by bryanharley** [comment]
I had this in my notes of things to look at.  I'd suggest when the custom post type 'show' is registered, a custom taxonomy 'cablecast_categories' should be registered too, rather than using WordPress' default category taxonomy.  

That way cablecast_categories will only apply to the "show" post type, and not the standard "post" post type. 

This would require changes in content.php to register the taxonomy, and sync.php so the Cablecast categories get synced to the new custom cablecast_categories taxonomy,.

**Comment by superlou** [comment]
Do you think people need the option of using the default post category? I can tentatively see an argument for blog-heavy sites to be able to intersperse their results between videos and standard posts. For my application, I just need the separate taxonomy, though.

**Comment by bryanharley** [comment]
Nah, I think it would be best to keep the Cablecast categories separate and connected to the "show" custom post type only.

**Comment by superlou** [comment]
It looks pretty straightforward, essentially changing all references to `category` to `show_category` and adding the custom taxonomy. That said, I'm having a world of hurt trying to get the old setup cleared out. What does `cablecast_sync_index` do?

**Comment by bryanharley** [comment]
I haven't implemented this change on my dev site yet, but to get rid of the old categories you might need to blow out the Cablecast database tables and re-sync.
